test/__snapshots__/github.test.ts.snap

```diff
@@ -370,7 +334,7 @@ jobs:
         uses: actions/download-artifact@v2
         with:
           name: cdk.out
-          path: cdk.out
+          path: temp/cdk.out
```

Comment:
This diff relates to the issue I've raised above. I don't think we want path: cdk.out anymore since that points to the base directory, not the temp directory. This previously wasn't an issue because we didn't actually create + add any files to cdk.out.